.docs/README.md

```diff
@@ -183,6 +184,43 @@ ispa.api:
 | getAll()      | /user       | GET  |
 | getById($id)  | /user/{$id} | GET  |
 
+## Adminus NMS
+
+### Configuration
+
+```yaml
+ispa.api:
+    app:
+        adminusNms:
```

Comment:
It must be named snake_case, because we have this kind of ENV variable.

`NETTE__ADMINUS_NMS`